packages/zoe/test/unitTests/contracts/test-operaConcertTicket.js

```diff
@@ -0,0 +1,409 @@
+                          start: termsOfBob.start,
+                          number: 2,
+                        },
+                        {
+                          show: termsOfBob.show,
+                          start: termsOfBob.start,
+                          number: 3,
+                        },
+                      ]),
+                    );
+
+                    const bobSecondProposal = harden({
+                      give: { Money: moola(2 * 22) },
+                      want: { Ticket: ticket2and3Amount },
+                    });
+                    const bobSecondPaymentForTicket = bobPurse.withdraw(
+                      moola(2 * 22),
+                    );
+
+                    return zoe
+                      .redeem(bobInvite2, bobSecondProposal, {
+                        Money: bobSecondPaymentForTicket,
+                      })
+                      .then(
+                        ({
+                          seat: { performExchange: performExchange2 },
+                          payout: payout2P,
+                        }) => {
+                          performExchange2();
+
+                          return payout2P.then(secondBobPayout => {
+                            return ticketIssuer
+                              .getAmountOf(secondBobPayout.Ticket)
+                              .then(bobSecondTicketAmount => {
+                                t.equal(
+                                  bobSecondTicketAmount.extent.length,
+                                  2,
+                                  'Bob should have received 2 tickets',
+                                );
+                                t.ok(
+                                  bobSecondTicketAmount.extent.find(
+                                    ticket => ticket.number === 2,
+                                  ),
+                                  'Bob should have received tickets #2',
+                                );
+                                t.ok(
+                                  bobSecondTicketAmount.extent.find(
+                                    ticket => ticket.number === 3,
+                                  ),
+                                  'Bob should have received tickets #3',
+                                );
+                              });
+                          });
+                        },
+                      );
+                  });
+                });
+              });
+            });
+        });
+    },
+  );
+
+  return Promise.all([contractReadyP, bobPartFinished])
+    .then(([{ publicAPI, _operaPayout, _complete }]) => {
+      // === Final Opera part ===
+      // getting the money back
+      const availableTickets = publicAPI.getAvailableTickets();
+
+      t.equal(availableTickets.length, 0, 'All the tickets have been sold');
+
+      const operaPurse = moolaIssuer.makeEmptyPurse();
+
+      const done = _operaPayout.then(payout => {
```

Comment:
We've been reserving underscores for variables that won't be used. Since we are using the operaPayout, can we rename?